includes/blocks/class-sensei-course-progress-block.php

```diff
@@ -44,12 +44,11 @@ public function register_block() {
 	 * @return string The HTML of the block.
 	 */
 	public function render_course_progress( $attributes ) : string {
-		if ( ! Sensei()->course::is_user_enrolled( get_the_ID() ) ) {
```

Comment:
I think the purpose was not showing the progress bar if the user is not enrolled. Do we really want this change?

Reply:
Oops this is a mistake, restored in 57fea28